web-console/old-console/index.html

```diff
@@ -42,7 +42,7 @@
         <a href="config.html">Configure Coordinator Parameters</a>
     </div>
     <div>
-        <a href="enable.html">Enable/Disable Datasources</a>
+        <a href="use-data-source.html">Mark all segments in data sources as used or unused</a>
```

Comment:
I think we should keep `Datasource` text as it is for consistency reasons with previous versions and documentation as conceptually I am used to think datasource as a noun which represents something similar to what table represents in database world. Also there are many places where `datasource` is used as a single word in documentation and new unified console. For example - https://druid.apache.org/docs/latest/design/index.html#datasources-and-segments says `Druid data is stored in "datasources", which are similar to tables in a traditional RDBMS.`